docs/internationalization/core.md

```diff
@@ -149,6 +149,23 @@ The translation strings for repairs issues are defined under the `issues` key. A
 
 ### Entities
 
+#### Name of entities
+Integrations can provide translations for names of its entities. To do this, provide an `entity` dictionary, that contains a translations of the names and set the entity's `translation_key` property to a key under a domain in the `entity` dictionary.
```

Comment:
"entity dictionary", is that in `strings.json`? If so it's not a dictionary but an object and we should clarify that it's done in `strings.json`.

Reply:
Yeah, it refers to the JSON object in `strings.json`, this PR reuses the existing wording from here https://developers.home-assistant.io/docs/internationalization/core

We should maybe improve what we already have first, and then update this PR?

Do you suggest that:
- We repeat in each section that backend translations are in `strings.json`
- We should use correct JSON terminology, e.g. "object" not "dictionary"

Reply:
I thought we were on the entity page. Here we don't need to mention `strings.json` again.

We should not talk about dictionary if we're describing a JSON file.

Reply:
"dictionary" is replaced by "object" in this PR: https://github.com/home-assistant/developers.home-assistant/pull/1712